content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,207 @@
+# Flare Architecture
+
+This section contains information about some parts of the Flare Network.
+In addition, some information regarding Avalanche Platform is also present, since Flare adopted some parts of the Avalanche in it's own ecosystem.
+
+## Avalanche Platform
+
+Avalanche is an open-source platform for launching decentralized applications and enterprise blockchain deployments in one interoperable, highly scalable ecosystem.
+Avalanche is the first decentralized smart contracts platform built for the scale of global finance, with near-instant transaction finality.
+Ethereum's developers can quickly build on Avalanche as Solidity works out-of-the-box.
+
+A key difference between Avalanche and other decentralized networks is the consensus protocol.
+The Avalanche protocol employs a novel approach to a consensus to achieve its strong safety guarantees, quick finality, and high-throughput without compromising decentralization.
+
+The purpose of Avalanche platform:
+
+* creation of subnets and blockchains tied to the main Avalanche network for general and corporate purposes;
+* creation and hosting of decentralized applications;
+* creation of smart assets (for example, new financial derivatives and DeFi solutions).
+
+### Subnets
+
+Everything on Avalanche is a _subnet_, and every chain is part of a subnet.
+There is a special subnet called the _Primary Network_, which validates Avalanche's built-in blockchains.
+Validators are required to be a member of the Primary Network; all other subnets are optional.
+Each subnet can have multiple blockchains just like the primary avalanche network.
+The Primary Network contains three build-in blockchains:
+
+* Platform Chain or P-Chain (manages the subnets, coordinates all the validator nodes and the staking mechanism)
+* Contract Chain or C-Chain (network for working with smart contracts)
+* Exchange Chain or X-Chain (meant for creation, management and transaction of tokens on the network – based on DAG a unique kind of consensus model)
+
+In total, the Avalanche ecosystem (as of the end of Decemner 2021) operates 13 subnets and 10 blockchains (three main plus 7 minor ones: Dtehereum, Btehereum, C and X-chain PoW and others).
+
+All the details regarding the operation of the platform, tutorials and tools are described in the [official Avalanche website](https://docs.avax.network/build/tutorials/platform/create-a-subnet/.)
+
+### Avalanche Consensus Protocol
+
+Avalanche Platform reach a consensus by using Avalanche Consensus Protocol.
+It has three mechanisms that give structural support to the network.
+These are the non-BFT protocol (Slush) that progressively built up to Snowflake and Snowball.
+These are single-decree consensus protocols with increasing robustness and are all based on the common majority-based metastable voting mechanism.
```

Comment:
```suggestion
The Avalanche platform uses the Avalanche Consensus Protocol.
It has three mechanisms that give structural support to the network:

* the non-BFT protocol (Slush) — that progressively built up to Snowflake and Snowball;
* single-decree consensus protocols with increasing robustness which are all based on the common majority-based metastable voting mechanism;
* <insert missing mechanism?>
```

I feel like this paragraph could gain from being restructured. Also you mention three mechanisms but unless I misunderstood what you meant, I only see two of them there.